components/eam/src/chemistry/modal_aero/modal_aero_drydep.F90

```diff
@@ -733,4 +615,380 @@ subroutine calcram(ncol,landfrac,icefrac,ocnfrac, &
+  real(r8) function air_kinematic_viscosity( temp, pres, rair )
+
+    real(r8),intent(in) :: temp   ! air temperature [K]
+    real(r8),intent(in) :: pres   ! air pressure [Pa]
+    real(r8),intent(in) :: rair   ! gas constant of air [J/K/kg]
+
+    real(r8) :: vsc_dyn_atm  ! kinematic viscosity of air [m2 s-1]
+    real(r8) :: rho          ! density of air [kg/m3]
+
+    vsc_dyn_atm = air_dynamic_viscosity( temp)
+    rho = pres/rair/temp
+
+    air_kinematic_viscosity = vsc_dyn_atm/rho
+
+  end function air_kinematic_viscosity
+
+  !======================================================
+  ! Slip correction factor [unitless]. 
+  ! See, e.g., SeP97 p. 464 and Zhang L. et al. (2001),
+  ! DOI: 10.1016/S1352-2310(00)00326-5, Eq. (3).
+  !======================================================
+  real(r8) function slip_correction_factor( dyn_visc, pres, temp, rair, pi, particle_radius ) 
+
+    real(r8),intent(in) :: dyn_visc         ! dynamic viscosity of air [kg m-1 s-1]
+    real(r8),intent(in) :: pres             ! air pressure [Pa]
+    real(r8),intent(in) :: temp             ! air temperature [K]
+    real(r8),intent(in) :: rair             ! gas constant of air [J/K/kg]
+    real(r8),intent(in) :: pi               ! constant pi = 3.14159....
+    real(r8),intent(in) :: particle_radius  ! particle radius [m]
+
+    real(r8) :: mean_free_path  ! [m]
+
+    mean_free_path = 2.0_r8*dyn_visc/( pres*sqrt( 8.0_r8/(pi*rair*temp) ) )
+
+    slip_correction_factor = 1.0_r8 + mean_free_path * &
+                             ( 1.257_r8+0.4_r8*exp(-1.1_r8*particle_radius/mean_free_path) ) / &
+                             particle_radius
+
+  end function slip_correction_factor
+
+  !====================================================================
+  ! Calculate the Schmidt number of air [unitless], see SeP97 p.972
+  !====================================================================
+  real(r8) function schmidt_number( temp, pres, radius, rair, pi, boltz, vsc_dyn_atm, vsc_knm_atm )
+
+    real(r8),intent(in) :: temp             ! air temperature [K]
+    real(r8),intent(in) :: pres             ! air pressure [Pa]
+    real(r8),intent(in) :: radius           ! particle radius [m]
+    real(r8),intent(in) :: rair             ! gas constant of air [J/K/kg]
+    real(r8),intent(in) :: pi               ! constant pi = 3.14159....
+    real(r8),intent(in) :: boltz            ! Boltzman's constant [J/K/molecule] 
+    real(r8),intent(in) :: vsc_dyn_atm      ! dynamic viscosity of air [kg m-1 s-1]
+    real(r8),intent(in) :: vsc_knm_atm      ! kinematic viscosity of air [m2 s-1]
+
+    real(r8) :: slp_crc   ! slip correction factor [unitless]
+    real(r8) :: dff_aer   ! Brownian diffusivity of particle [m2/s], see SeP97 p.474
+
+    slp_crc = slip_correction_factor( vsc_dyn_atm, pres, temp, rair, pi, radius )
+    dff_aer = boltz * temp * slp_crc / (6.0_r8*pi*vsc_dyn_atm*radius) 
+
+    schmidt_number = vsc_knm_atm / dff_aer
```

Comment:
There is also a calculation of Schmidt number in wet deposition (in `aero_model.F90`). The calculation of `slip_correction_factor` `(dumfuchs` in `aero_model.F90`) looks different. Not sure why they are different. I will note it in the comments.